garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -350,8 +350,8 @@ L.vis_desc        = [[
 Tatort-Visualisierungs-Gerät.
 
 Analysiere eine Leiche um zu sehen,
-wie die Leiche umgebracht wurde, aber nur,
-wenn sie erschossen wurde.]]
+wie der Spieler umgebracht wurde, funktioniert
```

Comment:
Please use "die Person" to make it consistent with the other parts of the translation.
See https://github.com/Facepunch/garrysmod-issues/issues/2818 for more information about this.